libs/sdk-ui-tests/stories/visual-regression/kit/EmbedInsightDialog/EmbedInsightDialogBase.tsx

```diff
@@ -79,6 +80,30 @@ export const EmbedInsightDialogBaseExamples: React.VFC = () => {
                         action("onOptionsChange");
                     }}
                     openSaveInsightDialog={action("openSaveInsightDialog")}
+                    showWebComponentsTab={true}
```

Comment:
Please move this prop to line 74, as props which are non-function should be next to each other, for the sake of consistency.